src/elements/ModuleSpecifier.js

```diff
@@ -12,6 +12,4 @@ export default class ModuleSpecifier extends Node {
         super(type, children);
         this.isModuleSpecifier = true;
     }
-
-    local: Object;
```

Comment:
If I don't remove this line `moduleSpecifier.local` gets set to `undefined` after the constructor runs